Components/navbar.js

```diff
@@ -21,23 +22,16 @@ const useStyles = makeStyles((theme) => ({
         background: "#9A1E1E"
     },
     logo: {
-        maxWidth: 60,
+        maxWidth: 70,
         padding: "20px 20px 20px 0"
     },
     menu_button: {
+        color: 'white',
         fontWeight: 'bold',
-    },
-    special_button: {
-        borderRadius: "1em",
-        backgroundColor: '#FFFFFF',
-        bottom: '20px',
-        right: '20px',
-        position: 'absolute',
-        color: '#9A1E1E',
-        fontWeight: 'bold',
-        height: '50px',
-        width: '150px'
-    },
+		fontSize: 20,
```

Comment:
This fontsize seems a little large. Definitely like how you spaced things out but I think should be shrunken a little bit

Reply:
Changed to 15